coreSDK/amd/.npmignore

```diff
@@ -5,6 +5,6 @@
 
 # ... but these files
 !package.json
-!bundle/aicore.js
```

Comment:
@jpiyali I changed from aicore to applicationinsights-core-js for consistency. I can change it back if you feel like we should stick with aicore. I can go either way. If we change it to applicationinsights-core-js, we'll need to update the npm-vsts feed

Reply:
should the file name be just core.js and not so large? I checked a few other existing public packages, names do not need to be reflective of package?